main-script.js

```diff
@@ -252,4 +252,64 @@ for (let i = 0; i < projectButton.length; i += 1) {
 
 for (let i = 0; i < btnCloseModal.length; i += 1) {
   btnCloseModal[i].addEventListener('click', openAndClose);
-}
\ No newline at end of file
+}
+
+const form = document.getElementById('contact-form');
+
+const formInput = document.querySelectorAll('form .input-control');
+
+for (let i = 0; i < formInput.length; i += 1) {
+  const message = document.createElement('small');
+  message.classList.add('error');
+  formInput[i].appendChild(message);
+}
+
+function showMessage(input, message, type) {
+  const msg = input.parentNode.querySelector('small');
+  msg.innerText = message;
+
+  input.className = type ? 'success' : 'error';
+  return type;
+}
+
+function showError(input, message) {
+  return showMessage(input, message, false);
+}
+
+function showSuccess(input) {
+  return showMessage(input, '', true);
+}
+
+function hasValue(input, message) {
+  if (input.value.trim() === '') {
+    return showError(input, message);
+  }
+  return showSuccess(input);
+}
+
+function validateEmail(input, requiredMsg, invalidMsg) {
+  if (!hasValue(input, requiredMsg)) {
+    return false;
+  }
+
+  const emailRegex = /^[a-z][a-z]+[a-z](?![A-z])/;
+
+  const email = input.value.trim();
+  if (!email.match(emailRegex)) {
+    return showError(input, invalidMsg);
+  }
+  return true;
+}
```

Comment:
- [ ] [OPTIONAL] It would be nice to create a new javascript fie for the validation functionality, this makes your code more readable.